_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -509,7 +509,9 @@
 	},
 /obj/machinery/requests_console{
 	name = "Detective Requests Console";
-	pixel_y = 30
+	pixel_y = 30;
+	department = "Detective";
+	departmentType = 5
```

Comment:
Request Consoles are able to print Shipping packages, however, the Detective's Office has no disposal bin in their office on MetaStation. Not sure if they need any, but it is a food for thought. If you want to make a full use of the request console, they will need one.

Reply:
Sounds good, I'll change that up and test it in the test commit